packages/server/src/util/entity-utils.ts

```diff
@@ -389,3 +274,23 @@ export class EntityController<T = any> {
   };
 
 }
+
+function formatColumnValue(val: ColumnValue, column: ColumnInfo) {
+  if (val === null) {
+    return val;
+  }
+  switch (column.type) {
+    case ColumnType.String:
+      return val.toString().toLowerCase();
```

Comment:
What is the reasoning behind doing `toLowerCase` here?  It seems that a query such as `column1 = 'AbC'` would fail because it would turn into `column1 = 'abc'`?